lib/util.js

```diff
@@ -394,6 +399,23 @@ function formatValue(ctx, value, recurseTimes, ln) {
     return ctx.stylize('null', 'null');
   }
 
+  const now = Date.now();
+  if (now > ctx.computeTime) {
+    if (now > ctx.computeTime + 1000 * 60 * 60 + 100) {
+      const { constructor, tag } = getIdentificationOf(value);
+      return ctx.stylize(`[${constructor || tag || 'Object'}]`, 'special');
+    }
+    if (now < ctx.computeTime + 1000 * 60 * 60) {
+      process.emitWarning('util.inspect takes very long.', {
+        code: 'INSPECTION_TIMEOUT',
+        detail: 'util.inspect() received an object that takes very long to ' +
+                'compute the output for. If it takes more than 100 more ' +
```

Comment:
Nit: `more than 100 more` -> `more than 100`?

Reply:
That has a different meaning though.

Reply:
I think that is not unambiguous enough. I see your point of improving the wording though.